src/timers.cc

```diff
@@ -127,7 +138,8 @@ double ProcessCPUUsage() {
 double ThreadCPUUsage() {
 // FIXME We want to use clock_gettime, but its not available in MacOS 10.11. See
 // https://github.com/google/benchmark/pull/292
-#if defined(CLOCK_THREAD_CPUTIME_ID) && !defined(BENCHMARK_OS_MACOSX)
+#if defined(CLOCK_THREAD_CPUTIME_ID) && !defined(BENCHMARK_OS_MACOSX) && \
```

Comment:
EMSCRIPTEN defines `CLOCK_THREAD_CPUTIME_ID` but doesn't provide `clock_gettime`? That seems weird... Apple only does that is to support targeting older platforms with `-mmacosx-version-min=<ver>` .